### PR TITLE
Smoothing out envs

### PR DESCRIPTION
**Pull request by mdarby**
Stuck on running tests in Docker. The `rails-test` container refuses to see `postgres` container though it's linked and pingable from said container.

### CODE REVIEW

File: bin/containers/postgresql.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    --link postgres:postgres \
+    -e OAUTH_APP_NAME=$OAUTH_APP_NAME \
+    -e OAUTH_APP_REDIRECT_URI=$OAUTH_APP_REDIRECT_URI \
+    -e OAUTH_APP_SECRET=$OAUTH_APP_SECRET \
+    -e OAUTH_APP_UID=$OAUTH_APP_UID \
+    -e POSTGRES_HOST=$POSTGRES_HOST \
+    -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD \
+    -e POSTGRES_USER=$POSTGRES_USER \
+    -e RAILS_ENV=$RAILS_ENV \
+    my-app rake db:migrate db:seed
```
Comment (carolynvs) [inline bin/containers/postgresql.sh]: This is your problem. Looks like you lost some of my most recent changes, so when the tests are run, the migration scripts aren't applied. That's what the CI build is complaining about.

Not sure what else may have been missed but this line should be `my-app rake db:migrate:reset db:seed`

**Comment by carolynvs** [comment]
It looks like the CI build doesn't run the steps in our deploy script and instead kind of does its own generic rails build?

**Comment by mdarby** [comment]
Yeah, the CI just runs `rspec`; no Docker involved.

File: Dockerfile [hunk 2]
```diff
@@ -35,9 +35,9 @@ RUN mkdir /my-app
 WORKDIR /tmp
 COPY Gemfile Gemfile
 COPY Gemfile.lock Gemfile.lock
-RUN bundle install --without development test
```
Comment (mdarby) [inline Dockerfile]: THIS. THIS LINE. THIS IS THE LINE.

Comment (carolynvs) [inline Dockerfile]: KILL IT WITH FIRE! :fire:

**Comment by mdarby** [comment]
All tests finally running and passing.

**Comment by carolynvs** [comment]
Looks like `bin/deploy latest|dev|image` etc was nuked?

**Comment by mdarby** [comment]
If you run `./bin/deploy` as is it will default to the `development` env. It really comes down to what the `RAILS_ENV` env var is set to.

**Comment by mdarby** [comment]
Rails its self follows this pattern (FYI :))

**Pull request merged**